app/views/components/gds/errorSummary.scala.html

```diff
@@ -21,8 +21,9 @@
 @(errors: Seq[FormError], errorFieldName: Option[String] = None)(implicit messages: Messages)
 
 @errorList = @{errors.map { error =>
+    val errorKey = error.key.replace(".", "_").replace('[', '_').replace("]", "")
```

Comment:
👍 Something we did not have in Movements ;-)

Reply:
I copied that from movements, once I had worked out why the error links weren't working for the 'ducr.ducr' field.